pkgs/applications/video/obs-studio/plugins/obs-aitum-multistream.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  curl,
+  obs-studio,
+  qtbase,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-aitum-multistream";
+  version = "1.0.7";
+
+  src = fetchFromGitHub {
+    owner = "Aitum";
+    repo = "obs-aitum-multistream";
+    tag = version;
+    hash = "sha256-TqddyTBRWLyfwYi9I0nQE8Z19YL2RwkZqUwi7F9XpwQ=";
+  };
+
+  # Fix FTBFS with Qt >= 6.8
+  prePatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace 'find_qt(COMPONENTS Widgets Core)' 'find_package(Qt6 REQUIRED COMPONENTS Core Widgets)'
```

```suggestion
      --replace-fail 'find_qt(COMPONENTS Widgets Core)' 'find_package(Qt6 REQUIRED COMPONENTS Core Widgets)'
```

avoids silent failures on bump